controllers/apply/awards-for-all/form.js

```diff
@@ -624,16 +626,10 @@ module.exports = function({
                         cy: ''
                     }),
                     get introduction() {
-                        const seniorFirstName = get(
-                            'seniorContactName.firstName'
-                        )(data);
-                        const seniorSurname = get('seniorContactName.lastName')(
-                            data
-                        );
-                        const seniorName =
-                            seniorFirstName && seniorSurname
-                                ? safeHtml`, <strong data-hj-suppress>${seniorFirstName} ${seniorSurname}</strong>`
-                                : '';
+                        const seniorName = getContactFullName(data, 'senior');
```

Comment:
I extracted this into a function as initially we repeated the senior contact name in the validation error message, though this subsequently changed. Keeping it as a function though as I can see this coming back in future.